pkg/routing/p2p.go

```diff
@@ -223,8 +223,13 @@ func bootstrapFunc(ctx context.Context, bootstrapper Bootstrapper, h host.Host)
 		bootstrapCtx, bootstrapCancel := context.WithTimeout(context.Background(), 10*time.Second)
 		defer bootstrapCancel()
 
+		// TODO (phillebaba): Consider if we should do a best effor bootstrap without host address.
+		hostAddrs := h.Addrs()
+		if len(hostAddrs) == 0 {
```

Comment:
This also causes problems, as closing the host above causes h.Addrs() to return nil.